lib/elixir/lib/task/supervised.ex

```diff
@@ -183,6 +193,13 @@ defmodule Task.Supervised do
     )
   end
 
+  defp get_callers do
+    case :erlang.get(:"$callers") do
+      [_ | _] = list -> [self() | list]
```

Comment:
Why add the task itself to the list?

Reply:
That's not the task, it is the caller of the task. :)

Reply:
Or to be precise, it is the caller of the task that will be created.

Reply:
Ah, makes sense. Sorry.